python/ray/serve/_private/common.py

```diff
@@ -376,6 +376,7 @@ class HTTPProxyStatus(str, Enum):
     STARTING = "STARTING"
     HEALTHY = "HEALTHY"
     UNHEALTHY = "UNHEALTHY"
+    INACTIVE = "INACTIVE"
```

Comment:
Does this need to be synced with KubeRay at all?

Reply:
I'm not finding a specific enum for it. But let me know if you think we need to make a change there as well

Reply:
If it's not in an enum, then we probably don't need to sync this with KubeRay. Thanks for checking!